### PR TITLE
Changelog for 1.19.3

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

File: web_src/js/features/codeeditor.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ const baseOptions = {
   overviewRulerLanes: 0,
   renderLineHighlight: 'all',
   renderLineHighlightOnlyWhenFocus: true,
-  renderWhitespace: 'none',
```
Comment (wxiaoguang) [inline web_src/js/features/codeeditor.js]: Curious, why it contains an unrelated change .....

Comment (techknowlogick) [reply web_src/js/features/codeeditor.js]: It was due to a mixup with vscode, and was overwritten by a force push to the 1.19 branch. I am attempting to cherry pick it back into the correct branch.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,18 @@ This changelog goes through all the changes that have been made in each release
...
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.19.3](https://github.com/go-gitea/gitea/releases/tag/1.19.3) - 2023-05-02
+
+* SECURITY
+  * Use golang 1.20.4 to fix CVE-2023-24539, CVE-2023-24540, and CVE-2023-29400
+* ENHANCEMENTS
+  * Enable whitespace rendering on selection in Monaco (#24444) (#24485)
+  * Improve milestone filter on issues page (#22423) (#24440)
+* BUGFIXES
```
Comment (yardenshoham) [inline CHANGELOG.md:14]:
```suggestion
* BUGFIXES
  * Fix api error message if fork exists (#24487) (#24493)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,18 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.19.3](https://github.com/go-gitea/gitea/releases/tag/1.19.3) - 2023-05-02
```
Comment (yardenshoham) [inline CHANGELOG.md]:
```suggestion
## [1.19.3](https://github.com/go-gitea/gitea/releases/tag/1.19.3) - 2023-05-03
```

**Review by yardenshoham — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Pull request merged**